components/data-table/__examples__/advanced-single-select-fixed-header.jsx

```diff
@@ -23,43 +56,16 @@ class Example extends React.Component {
 		sortColumnDirection: {
 			opportunityName: 'asc',
 		},
-		items: [
-			{
-				id: '8IKZHZZV80',
-				opportunityName: 'Acme - 1,200 Widgets',
-				accountName: 'Acme',
-				closeDate: '4/10/15',
-				stage: 'Value Proposition',
-				confidence: '70%',
-				amount: '$25,000,000',
-				contact: 'jrogers@acme.com',
-			},
-			{
-				id: '5GJOOOPWU7',
-				opportunityName: 'Acme - 200 Widgets',
-				accountName: 'Acme',
-				closeDate: '1/31/15',
-				stage: 'Prospecting',
-				confidence: '30%',
-				amount: '$5,000,000',
-				contact: 'bob@acme.com',
-			},
-			{
-				id: '8IKZHZZV81',
-				opportunityName: 'salesforce.com - 1,000 Widgets',
-				accountName: 'salesforce.com',
-				closeDate: '1/31/15 3:45PM',
-				stage: 'Id. Decision Makers',
-				confidence: '60%',
-				amount: '$25,000',
-				contact: 'nathan@salesforce.com',
-			},
-		],
+		items,
 		selection: [],
+		disabledSelection: [items[0]],
 	};
 
 	handleChanged = (event, data) => {
```

Comment:
Does the event fire on a disabled checkbox?

Reply:
I'm asking about this line. 

I'm thinking that disabled doesn't mean a row can't be selected. You could force select a row (that is make it selected) and then disable it. 

```
const selections = data.selection.filter(
			(s) => !this.state.disabledSelection.includes(s)
		);
```

Reply:
yes. Makes sense.  A row can be preselected but disabled.